src/release.js

```diff
@@ -42,6 +42,24 @@ module.exports = async function ({ github, context, inputs }) {
 
   const { opticUrl, npmTag, version, id } = releaseMeta
 
+  try {
+    const { data: draftRelease } = await github.rest.repos.getRelease({
+      owner,
+      repo,
+      release_id: id,
+    })
+
+    if (!draftRelease || !draftRelease.draft) {
```

Comment:
we should not check if the release is draft, because this would make this non-idempotent, meaning that we can't run it twice if anything breaks. it is enough to check if a release is there, if it is, we're good, even if it's a published release (remember we _explicitly_ talked about this)